Makefile

```diff
@@ -1,6 +1,8 @@
 include .env
 BINARY := oauth2_proxy
 VERSION := $(shell git describe --always --dirty --tags 2>/dev/null || echo "undefined")
+# Allow to override image repository.
+HUB ?= quay.io/pusher
```

Comment:
nit: I would have used `REGISTRY` rather than `HUB`.

Reply:
OK, done in 29f3b96. Thank you for the review!